src/AppBundle/Resources/config/content_view.yml

```diff
@@ -3,17 +3,28 @@ frontend_group:
         full:
             ng_category:
                 template: "@ezdesign/content/full/ng_category.html.twig"
-                controller: "ngsite.controller.full_view:viewNgCategory"
+                controller: "ngsite.controller.check_redirect"
                 match:
                     Identifier\ContentType: ng_category
+                queries:
+                    subtree:
+                        query_type: 'SiteAPI:Location/Subtree'
+                        max_per_page: '@=content.getFieldValue("page_limit").value > 0 ? content.getFieldValue("page_limit").value : viewParam("childrenLimit", 12)'
+                        page: '@=request.query.get("page", 1)'
+                        parameters:
+                            content_type: '@=split(content.getFieldValue("children_class_filter_include").text, ",")'
+                            visible: true
+                            exclude_self: true
+                            relative_depth: '@=content.getFieldValue("fetch_subtree").bool ? null : 1'
+                            sort: '@=location.parent.innerLocation.getSortClauses()'
```

Comment:
Shouldn't we sort by the current Location, not the parent one?

Reply:
@pspanja Yes, sorry. Fixed in https://github.com/netgen/media-site/pull/51/commits/b8b54927e380151577f63f3dbc37446803634868